taxonomies/ingredients.txt

```diff
@@ -64101,6 +65082,7 @@ nl:gepasteuriseerd ei van hennen met vrije uitloop
 en:barn eggs
 de:Eier aus Bodenhaltung
 es:huevos de granero
+fi:Maalaismunat
```

Comment:
```suggestion
fi:lattiakanojen munat, vapaan kanan munat, onnellisen kanan munat, pesämunat
```
Maalaismunat is a marketing term generally referring to caged hen's eggs.

Reply:
Thank you @Ban3 , I'll update it. (sadly the suggestions cannot be commited through github's web page directly as the file is too large)